### ISSUE TITLE
Selected place in the file manager is incorrect

### ISSUE THREAD

**Issue by aubreybourke** [elementary/files#919]
In the file manager when you select a place on the left hand side menu, it highlights it in gray. However the outline is incorrect. It should be a flat outline, but it is indented.

However, if you go to save a file, the file chooser displays the selected place correctly.

**Comment by cassidyjames** [comment]
@aubreybourke I'm having a hard time understanding what you're pointing out. Could you provide screenshots demonstrating the issue?

**Comment by aubreybourke** [comment]
Yes here you see in the file manager Documents has a white bottom-border

**Comment by aubreybourke** [comment]
Where as in the save file dialog it looks correct. For example

**Comment by cassidyjames** [comment]
@aubreybourke it looks like you tried attaching screenshots to an email? GitHub doesn't support that; you can visit the issue in your web browser and edit your comments to include screenshots.
